### PR TITLE
Automatically build and publish the container image for bootstrapping Gentoo Prefix

### PR DESCRIPTION
**Pull request by bedroge**
This action builds a multi-arch Docker image and pushes it to the Github Container Registry.
I've also adjusted our Ansible role, so that it pulls the image from the GCR.

### CODE REVIEW

**Comment by bedroge** [comment]
The tests are failing due to an issue with one of our workflows, this is fixed in #97.

File: .github/workflows/build-and-publish-bootstrap-container.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Convert and store repository owner in lowercase
+        run: |
+          echo REPOSITORY_OWNER=$(echo ${{ github.repository_owner }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
```
Comment (boegel) [inline .github/workflows/build-and-publish-bootstrap-container.yml:44]: I would use double quotes here, which makes this a bit easier to parse (for humans):

```suggestion
          echo "REPOSITORY_OWNER=$(echo ${{ github.repository_owner }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@bedroge Do you understand why CI is failing now?

**Comment by bedroge** [comment]
These issues have been fixed in another PR, so this PR probably needs to be synced with `main`.

**Comment by boegel** [comment]
@bedroge CI is still failing :-/

**Comment by bedroge** [comment]
```
/tmp/gentoo/var/db/repos/eessi/sys-cluster/reframe/reframe-3.6.2.ebuild", " * Reason: Filesize does not match recorded size", " * Got: 969", " * Expected: 1040"
```

Looks like we added a wrong checksum...

**Comment by bedroge** [comment]
CI for the installation playbook is broken, see #111, let's fix that later (not really related to anything from this PR anyway)...

**Review by boegel — APPROVED** [review]

**Pull request merged**